app/helpers/application_helper/button/instance_reconfigure.rb

```diff
@@ -1,10 +1,6 @@
 class ApplicationHelper::Button::InstanceReconfigure < ApplicationHelper::Button::Basic
-  def calculate_properties
-    super
-    self[:title] = @record.unsupported_reason(:resize) if disabled?
-  end
-
   def disabled?
-    !@record.supports_resize?
```

Comment:
[GenericFeatureButtonWithDisable](https://github.com/ManageIQ/manageiq/blob/master/app/helpers/application_helper/button/generic_feature_button_with_disable.rb) can be used here